requirements.txt

```diff
@@ -1,5 +1,5 @@
 pypdf2~=1.26.0
-pypdfium~=0.0.15
+pypdfium2~=0.4.0
```

Comment:
Thanks for merging. As this PR was created some time ago, you might want to update the requirement from `0.4.0` to `0.11.0`.